SpatialGDK/Source/SpatialGDKEditor/Private/Utils/TransientUObjectEditor.cpp

```diff
@@ -21,15 +22,18 @@ namespace
 	// Copied from FPropertyEditorModule::CreateFloatingDetailsView.
 	bool ShouldShowProperty(const FPropertyAndParent& PropertyAndParent, bool bHaveTemplate)
 	{
-		const UProperty& Property = PropertyAndParent.Property;
+		const GDK_PROPERTY(Property)& Property = PropertyAndParent.Property;
 
 		if (bHaveTemplate)
 		{
+#if ENGINE_MINOR_VERSION <= 24
 			const UClass* PropertyOwnerClass = Cast<const UClass>(Property.GetOuter());
+#else
+			const UClass* PropertyOwnerClass = Property.GetOwner<const UClass>();
```

Comment:
Isn't the outer different to the owner?

Reply:
I think this is taken from the updated version of the function this is copied from in the Engine.

Reply:
`GetOuter` is for `UObjects`. In 4.25, `FProperty` is not a `UObject` and instead it has `GetOwner` which does pretty much the same thing.

Reply:
@joshuahuburn I think you may be thinking about the UActorComponent's `GetOwner` which is different and unrelated to this.